Cargo.lock

```diff
@@ -5205,9 +5205,9 @@ dependencies = [
 
 [[package]]
 name = "sysinfo"
-version = "0.29.2"
+version = "0.29.11"
```

Comment:
Is this one was intentional?

Reply:
Not really but does it really hurt?

Reply:
This change is kind a out of scope for this PR (as the title and commit says 'update sysinfo to 0.30.1'). It would be more appropriate to revert this change I think (and update it in seperate commit if needed).